src/SDKs/ApiManagement/Management.ApiManagement/Microsoft.Azure.Management.ApiManagement.csproj

```diff
@@ -0,0 +1,17 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <Import Project="$([MSBuild]::GetPathOfFileAbove('AzSdk.reference.props'))" />
+  <PropertyGroup>
+    <PackageId>Microsoft.Azure.Management.ApiManagement</PackageId>
+    <Description>Provides ApiManagement management capabilities for Microsoft Azure.</Description>
+    <AssemblyTitle>Microsoft Azure API Management Management</AssemblyTitle>
+    <AssemblyName>Microsoft.Azure.Management.ApiManagement</AssemblyName>
+    <VersionPrefix>1.0.0-preview</VersionPrefix>
+    <PackageTags>Microsoft Azure ApiManagement management;API Managementt;REST HTTP client;azureofficial;windowsazureofficial;netcore451511</PackageTags>
+  </PropertyGroup>
+  <PropertyGroup>
+    <TargetFrameworks>net452;netstandard1.4</TargetFrameworks>
+  </PropertyGroup>
+  <ItemGroup>
+    <Folder Include="Customization\Models\" />
```

Comment:
@solankisamir you don't need to include any folders. Project system will include it. This is not needed